data/migrations/20191111130723_create-tables.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex) {
+  return knex.schema
+    .createTable('user', project => {
+      project.increments();
+      project.string('username', 100).notNullable();
+      project.text('password', 255).notNullable();
+    })
```

Comment:
When creating a table 'user' or ('users'), a better and clearer parameter for the callback function would be 'user'
So we would have user,increments(); user.string('username', 100)....

using 'project' might be misleading.